src/HealthChecks.Dapr/HealthChecks.Dapr.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
+    <PackageTags>$(PackageTags);Dapr</PackageTags>
+    <Description>HealthChecks.Dapr is the health check package for Dapr.</Description>
+    <VersionPrefix>$(HealthCheckDapr)</VersionPrefix>
```

Comment:
This time, I used the correct tag - it's `VersionPrefix` and **not** `Version` (bcc: @sungam3r)